README.md

```diff
@@ -1,27 +1,32 @@
-[![Actions Status](https://github.com/nickdesaulniers/github_actions_playground/workflows/Clang%20Linux%20CI%20v2/badge.svg)](https://github.com/nickdesaulniers/github_actions_playground/actions)
+[![Actions Status](https://github.com/clangbuiltlinux/continuous-integration2/workflows/Clang%20Linux%20CI%20v2%20next/badge.svg)](https://github.com/nickdesaulniers/github_actions_playground/actions?query=workflow%3A"Clang+Linux+CI+v2+next")
+[![Actions Status](https://github.com/clangbuiltlinux/continuous-integration2/workflows/Clang%20Linux%20CI%20v2%20mainline/badge.svg)](https://github.com/nickdesaulniers/github_actions_playground/actions?query=workflow%3A"Clang+Linux+CI+v2+mainline")
 
 Testing using [TuxBuild](https://gitlab.com/Linaro/tuxbuild) to build the Linux
 kernel with LLVM under CI.
 
+All test parameters are encoded in `generator.yml`; new trees, architectures,
+configs, etc. should be added there.
+
 ### Usage
 
+The tuxbuild and github actions workflow configs should be updated when
+`generator.yml` changes. Ex.
+```sh
+$ BRANCH=next
+$ ./generate_tuxbuild.py $BRANCH < generator.yml > $BRANCH.tux.yml
+$ ./generate_workflow.py $BRANCH < generator.yml > .github/workflows/$BRANCH.yml
+```
```

Comment:
Might be nice to eventually code this in a little script in the repo? Maybe even plug into a GitHub Actions workflow to make sure that all files are updated when `generator.yml` is changed?

Reply:
sure, generate all the things? Files #11